VSIX/MarkdownProvider/MarkdownProvider.csproj

```diff
@@ -27,6 +27,11 @@
   </PropertyGroup>
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
+    <StartAction>Program</StartAction>
```

Comment:
This part is needed so that others can F5 to run the VSIX since launch information is stored in the .user file which isn't committed to github